src/main/java/frc/robot/RobotContainer.java

```diff
@@ -273,8 +274,8 @@ private void configureButtonBindings() {
       }
 
       if (outtakeCommand != null && CDSForwardCommand != null) {
-        buttons2[Constants.RBumper].whileHeld(CDSForwardCommand);
-        buttons2[Constants.RTriggerButton].whileHeld(outtakeCommand);
+        buttons[Constants.RTriggerButton].whileHeld(CDSForwardCommand);
```

Comment:
Keep this on operator controller (buttons2) before merging to main.

Reply:
got it